trunk8.js

```diff
@@ -208,7 +208,11 @@
 		}
 		else if (!isNaN(width)) {
 			bite_size = length - width;
-
+			
+			if ( bite_size <= 0) {
```

Comment:
Remove the space before bite_size and change the <= to <. eatStr has a check for == 0.